config/thoth_demo.yaml

```diff
@@ -0,0 +1,6 @@
+---
+repositories:
+  - slug: saisankargochhayat/kebechet_sample
+    token: "{KEBECHET_TOKEN}"
+    service_type: github
+    service_url: https://github.com/saisankargochhayat/kebechet_sample
```

Comment:
Should this be removed and just the slug name, token and service type be sufficient? 
What about the other parameters? Should we attempt reading them from the repo's .thoth.yaml?

Reply:
`tls_verify` should not be taken into account in Kebechet to avoid man-in-the-middle-attacks.

`service_url` is not needed - it can be constructed out of `slug` and `service_type` in this case. For internal GitLab instances, we could keep `service_url` indeed (to reach internal gitlab instance, not the public one).